frontend/templates/product/MetaTags.tsx

```diff
@@ -52,6 +52,15 @@ export function MetaTags({ product, selectedVariant }: MetaTagsProps) {
          <link rel="canonical" href={canonicalUrl} />
          <meta property="og:url" content={canonicalUrl} />
 
+         {product.enabledDomains?.map((store) => (
```

Comment:
Hmm... Looks like this will also include the current store. I think that's bad cause it'll be like a second canonical. Should we filter that out on the php app side, or try to here?

Reply:
I agree that we should probably do it in PHP.

I didn't realize we needed to exclude the current store, because I saw this sort of self-referencing on the EU store.

![image](https://user-images.githubusercontent.com/52104630/204367128-f915cc8d-a725-47d1-ae2e-1bde039c4245.png)

https://eustore.ifixit.com/products/iphone-6s-replacement-battery